### DISCUSSION TITLE
WindowTop translations

### DISCUSSION THREAD

**Discussion by gileli121** [WindowTop/WindowTop-App#323] [General]
Currently, the supported languages other than English are:

* German
* French
* Italian
* Korean
* Portuguese
* Swedish
* Chinese

All translations are contributed freely by users who suggest translating the app into their language.

If you want to contribute a translation, contact me on gileli121@gmail.com or via https://windowtop.info/contact/ 

If you want to report a translation mistake, you can do so here by providing a screenshot of the mistake and a fixed translation.

Thank you.

**Comment by gileli121** [comment]
WindowTop is being translated into Japanese and Turkish
Thanks a lot, to daisuke for Japanese translation.
and for Doğukan for the Turkish translation


The new translation is currently available in beta
https://github.com/WindowTop/WindowTop-App/releases/tag/v5.22.5

**Reply by boybh** [reply]
Can it be translated into Vietnamese language?

**Reply by gileli121** [reply]
@boybh 
Yes, please contact me via email contact@windowtop.info
And I will provide you with instructions for the translation

**Reply by Jefri2003** [reply]
hello I would like to contribute to the translation into Spanish I use this program a lot and I am sorry that it is not in Spanish

**Reply by gileli121** [reply]
@Jefri2003
Please contact me through the contact email, and I will provide you with an explanation on how to translate the software.  
Thank you very much!

**Reply by DefftoN** [reply]
for Ukrainian at least one mistake, in the scrawl instead of the letter "ы" it should be "і", also the "Anchors" menu is not completely clear translated, it is better to make "Прив’язки" instead of "Анкери".
<img width="1116" height="790" alt="33333333" src="https://github.com/user-attachments/assets/2c300fa1-6d6a-449f-b189-07d33815a066" />

**Comment by lennyqebian** [comment]
Hey Gil, I just sent an email to [contact@windowtop.info](mailto:contact@windowtop.info). I would like to support you in a translation into Spanish, on the lookout for instructions.
